src/emittermixin.js

```diff
@@ -150,14 +156,18 @@ const EmitterMixin = {
 		//     ...
 		// }
 
-		if ( !( emitters = this._listeningTo ) ) {
-			emitters = this._listeningTo = {};
+		if ( !this[ _listeningTo ] ) {
+			this[ _listeningTo ] = {};
 		}
 
-		if ( !( emitterId = emitter._emitterId ) ) {
-			emitterId = emitter._emitterId = uid();
+		emitters = this[ _listeningTo ];
+
+		if ( !emitter._emitterId ) {
+			emitter._emitterId = uid();
```

Comment:
So, to sum up, `Symbol` is used to save the property in the object, then "non-symbol" getter and setter are used to access the property.